src/app/components/pages/send-skycoin/send-verify/send-verify.component.ts

```diff
@@ -47,9 +52,11 @@ export class SendVerifyComponent {
   }
 
   private onError(error) {
+    const errorMessage = parseResponseMessage(error['_body']);
```

Comment:
`error['_body']` is undefined, so an error occurs when reaching this line.

To test the errors, please modify the last line of `WalletService.injectTransaction` with `return Observable.throw(new Error("Error"));`

Reply:
Fixed